config/serverless-operator.yaml

```diff
@@ -12,6 +12,16 @@ config:
         version: "4.15"
       - onDemand: true
         version: "4.13"
+    release-1.35:
+      prowgen:
+        enabled: false
+      konflux:
+        enabled: false
+        imageOverrides:
```

Comment:
It is here so that Konflux generator for other repositories (Eventing, etc.) can read it from here.